recipes/subarch-select/meta.yaml

```diff
@@ -0,0 +1,52 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 16 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "subarch-select" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/ekg/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: acb41a936637d7488997646f61f386f1869bd91aa093e0f48d6299c95c9f9851
+    patches:
+      - makefile_tweaks.patch
+
+build:
+  skip: True  # [win or osx]
+  script: |
+    make CPPFLAGS="-DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -isystem $BUILD_PREFIX/include"
```

Comment:
Probably this is better placed in a build.sh. You should be passing in here CPPFLAGS=$CPPFLAGS, and do the same for CXXFLAGS.

Reply:
I moved this to a simplified `build.sh`, thanks